pyproject.toml

```
@@ -4,7 +4,11 @@ version = "2.2.1"
 description = "A set of helper functions for CSV to Salesforce procedures, with reporting in AWS S3"
 readme = "README.md"
 repository = "https://github.com/Kicksaw-Consulting/kicksaw-integration-utils"
-authors = ["Alex Drozd <drozdster@gmail.com>", "Tim Sabat <tim@kicksaw.com>"]
+authors = [
+    "Alex Drozd <drozdster@gmail.com>",
```

```suggestion
    "Alex Drozd <alex@kicksaw.com>",
```
let's get my personal email outta here 😆